src/main/java/com/fauna/client/Stats.java

```diff
@@ -0,0 +1,59 @@
+package com.fauna.client;
+
+public final class Stats {
```

Comment:
This could use more javadoc. At least how this is supposed to be used.

Reply:
Oh, excellent point. I had blinders on for some reason